tests/test_fixtures/rule_based_profiler/example_notebooks/DataAssistants_Instantiation_And_Running-OnboardingAssistant-Spark.ipynb

```diff
@@ -0,0 +1,1498 @@
+    "\n",
+    "Other values for cardinality values include: \n",
+    "\n",
+    "```python \n",
+    "ZERO = AbsoluteCardinalityLimit(\"ZERO\", 0)\n",
+    "ONE = AbsoluteCardinalityLimit(\"ONE\", 1)\n",
+    "TWO = AbsoluteCardinalityLimit(\"TWO\", 2)\n",
+    "VERY_FEW = AbsoluteCardinalityLimit(\"VERY_FEW\", 10)\n",
+    "FEW = AbsoluteCardinalityLimit(\"FEW\", 100)\n",
+    "SOME = AbsoluteCardinalityLimit(\"SOME\", 1000)\n",
+    "MANY = AbsoluteCardinalityLimit(\"MANY\", 10000)\n",
+    "VERY_MANY = AbsoluteCardinalityLimit(\"VERY_MANY\", 100000)\n",
+    "UNIQUE = RelativeCardinalityLimit(\"UNIQUE\", 1.0)\n",
+    "\n",
+    "... # and more\n",
+    "```\n",
+    "\n",
+    "The full list of cardinality values used by Great Expectations can be found in the [great_expectations repo here](https://github.com/great-expectations/great_expectations/blob/da2376b613843844afc041538269bb7683444b1f/great_expectations/rule_based_profiler/helpers/cardinality_checker.py#L55). \n",
+    "\n",
+    "The three Expectations each have their own parameters\n",
+    "\n",
+    "`expect_column_values_to_be_in_set` requires `column` (column name) and `value_set`, which is calculated for all columns that meet the cardinality threshold. \n",
+    "\n",
+    "`expect_column_unique_value_count_to_be_between` has `column` (column name) and `min_value` and `max_value`.\n",
+    "\n",
+    "Estimation will be done using the `exact` estimator by default, but if you select `drop_outliers` then the following parameters are set by default for `bootstrap` estimation\n",
+    "\n",
+    "* `n_resamples` : For bootstrapping. It is set by default to `9999` which is the default in https://docs.scipy.org/doc/scipy/reference/generated/scipy.stats.bootstrap.htm\n",
+    "* `false_positive_rate`: user-configured fraction between 0 and 1 expressing desired false positive rate for\n",
+    "    identifying unexpected values as judged by the upper- and lower- quantiles of the observed metric data. Default is `0.05`.\n",
+    "* `random_seed`: Seed for randomization. If omitted (which is the default), then we use `np.random.choice`. otherwise, we use `np.random.Generator(np.random.PCG64(random_seed))`.\n",
+    "\n",
+    "`expect_column_proportion_of_unique_values_to_be_between` has the following parameters\n",
+    "   \n",
+    "* `column`: column name\n",
+    "* `min_value`:  minimum proportion of unique values, ranging from 0 to 1\n",
+    "* `max_value`:  minimum proportion of unique values, ranging from 0 to 1\n",
+    "* `strict_min`: determine whether the minimum proportion of unique values must be strictly greater than min value, with the `default=False`\n",
+    "*`strict_max`: determine whether the minimum proportion of unique values must be strictly smaller than max value, with the `default=False`\n",
```

Comment:
nit: you need a space after `*` or this renders funny.

Reply:
Done!